PrebidMobile/PrebidServerAdapter/src/main/java/org/prebid/mobile/prebidserver/PrebidServerAdapter.java

```diff
@@ -514,9 +519,30 @@ private JSONObject getUserObject() {
             if (!TextUtils.isEmpty(finalKeywords)) {
                 user.put("keywords", finalKeywords);
             }
+            JSONObject ext = new JSONObject();
+            JSONArray consentStrings = new JSONArray();
+            for (String s : TargetingParams.getGDPSConsentStrings(context)) {
+                consentStrings.put(s);
+            }
+            user.put("ext", ext);
         } catch (JSONException e) {
         }
         return user;
     }
+
+    private JSONObject getRegsObject(Context context) {
+        JSONObject regs = new JSONObject();
+        try {
+            JSONObject ext = new JSONObject();
+            if (TargetingParams.isUnderGDPR(context)) {
```

Comment:
what happens if the GDPR is not set by the publisher? by default it should be sent as true not false